.all-contributorsrc

```diff
@@ -0,0 +1,4 @@
+{
+  "projectName": "data-safe-haven",
+  "projectOwner": "alan-turing-institute"
+}
```

Comment:
There are quite a lot of configuration options we should look at.

In particular, can we make sure that the all contributors table is also added somewhere in the docs?

[Here](https://github.com/sa-tre/satre-specification/blob/main/.all-contributorsrc) is an example.

Reply:
@JimMadge I've added a page on readthedocs too. 

Can you take a look whether you think there are any other options you'd like to configure?

Also, in terms of contributions, I have just put "code" and "doc" and guessed when people were likely to have been doc editors only - might be worth taking a look at the table and seeing if you think modifications should be made